lasy/laser.py

```diff
@@ -169,10 +209,16 @@ def propagate(self, distance, nr_boundary=None, backend="NP", show_progress=True
         distance : scalar
             Distance by which the laser pulse should be propagated
 
+        initial_optical_element: an :class:`.OpticalElement` object (optional)
+            Represents a thin optical element, through which the laser
+            propagates, before propagating for `distance` in free space.
+            If this is `None`, no optical element is used.
+
```

Comment:
Should this be removed?
```suggestion
```

Reply:
Ah, yes, sorry. I thought I had removed it.